hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -162,6 +176,7 @@ public Response getKeysForContainer(
     Map<String, KeyMetadata> keyMetadataMap = new LinkedHashMap<>();
     long totalCount;
     try {
+      prevKeyPrefix = correctPathForLayout(prevKeyPrefix);
```

Comment:
IMO, we do not need correctPathForLayout, as this input is part of pagination.
ie. - with pagesize 10,
- get list with pageSize 10, and key as empty. -- list first 10 element
- get list with pagesize 10, and key as last element -- list next 10 element

Reply:
Thanks for the analysis sumit, you were absolutely right on this one this one the prevKey Prefix was used for pagination purposes.